docs/src/references/api/python/calculators.rst

```diff
@@ -35,6 +35,11 @@ Available Calculators
     :show-inheritance:
 
 
+.. autoclass:: rascaline.power_spectrum.PowerSpectrum
+    :members:
+    :show-inheritance:
+
+
```

Comment:
I don't know if we want to document this in the same way as the other calculators, since it does not support the same set of options. Maybe in a separate place?